packages/esm-patient-flags-app/translations/es.json

```diff
@@ -19,6 +20,7 @@
   "flagEnabledSuccessfully": "Bandera activada con éxito",
   "flagEnableError": "Error al activar la bandera",
   "loading": "Cargando",
+  "matchesForSearchTermfalsemany": "{count} flag{plural}",
```

Comment:
@vasharma05 could you look into why these specific translations with keys and strings that use pluralization change so often? I suspect it might require looking into upgrading `i18next`, `react-i18next` and possibly even i18next-parser to the latest available versions to leverage potential upstream API improvements.

Reply:
Sure @denniskigen , I'll be on it

Reply:
I believe my approach for `{plural}` is wrong in here, since in `en`, appending `s` or `es` works, but in other languages, this is not true.

Reply:
It doesn't even work for English, e.g., "injury" -> "injuries". "Injuryes" was a reasonable form in the 15th century, but has fallen out of favor.

Reply:
This will be improved in #1413 .